apps/builddao/widget/components/navigation/Navbar.jsx

```diff
@@ -0,0 +1,450 @@
+const { Button } = VM.require("buildhub.near/widget/components") || {
```

Comment:
Let's just move this to components.Navbar

Reply:
Ok